platform/ios/app/MBXViewController.m

```diff
@@ -40,6 +40,7 @@ + (void)initialize
             @"MBXUserTrackingMode": @(MGLUserTrackingModeNone),
             @"MBXShowsUserLocation": @NO,
             @"MBXDebug": @NO,
+            @"MGLMapboxMetricsTestServerURL": @"https://cloudfront-staging.tilestream.net",
```

Comment:
On second though, PROGRESS++! Changed to `MGLTelemetryTestServerURL` in ff425b9.